### ISSUE TITLE
`pre-commit autoupdate` always fails if detect-secrets is present

### ISSUE THREAD

**Issue by ealter** [Yelp/detect-secrets#87]
```
hookid: detect-secrets                

The supplied baseline may be incompatible with the current                  
version of detect-secrets. Please recreate your baseline to                 
avoid potential mis-configurations.   

Current Version: 0.10.3               
Baseline Version: 0.9.1
```

detect-secrets should heal the repository when this happens rather than requiring manual intervention.

**Comment by asottile** [comment]
this is the only hook I know of that does this, it's a wart to the ecosystem.  The expectation is that `autoupdate` is generally an automatic process.

**Comment by KevinHock** [comment]
This should be fixed in the latest version (`11.0` via https://github.com/Yelp/detect-secrets/pull/96) 👍 

Sorry for the late reply.
